### DISCUSSION TITLE
So great, aws-sdk for java within a week!

### DISCUSSION THREAD

**Discussion by boaks** [eclipse/dash-licenses#180] [Show and tell]
This tool is really, really great!

Compared to the process years ago, it makes even larger third-party dependencies very easy.

I start the license check a week ago, and I got a lot of restrictions for the aws-sdk.
A week later, the most restrictions are gone and only two were left.
With the automatic CQs these two are then solved with hours.

**Comment by waynebeaton** [comment]
There's still some challenges, and lots of work to do, but I'm generally happy with what we've managed to produce so far.

In the process, we did make a couple of changes to the process. Removing the PMC from the equation smoothed things over a bit as -- even when the PMC was very responsive -- waiting for them always added some time to the process.

Having the ability to tweak the automatic processor on the fly (that is, letting me flex my programming skills) has also been handy and has resulted in us being able to automatically review and approve a lot of content consistently and confidently.

We've also been able to collect a lot more information from the scanner directly into the issue which makes it easier and faster for the IP Team to review the requests that require manual intervention.

The Eclipse Dash License Tool started out as an experiment. I think that we can declare that experiment successful. But, like I said, there's more work to do.
